app/uk/gov/hmrc/gform/models/FieldValueMaker.scala

```diff
@@ -44,6 +44,13 @@ class FieldValueMaker(json: JsValue) {
     format.map(FormatParser.validate).sequence
   }
 
+  lazy val presentationHint: Option[String] = (json \ "presentationHint").asOpt[String]
```

Comment:
For such json `{"presentationHint" : 123}` presentation hint will become `None`. What we need is to have`Left("expected String but got int")`.